static/app/views/dashboards/widgetBuilder/components/widgetBuilderSlideout.tsx

```diff
@@ -56,6 +56,7 @@ function WidgetBuilderSlideout({
   const organization = useOrganization();
   const {state} = useWidgetBuilderContext();
   const [initialState] = useState(state);
+  const [error, setError] = useState<Record<string, any>>({});
```

Comment:
Wish there was a better way to type this error object, but I think this is okay for now. If we find something better to type this with then we should look at making it more explicit